vue-app/src/components/RoundInformation.vue

```diff
@@ -207,20 +207,20 @@
                 ><img style="opacity: 0.6" width="16px" src="@/assets/info.svg"
               /></tooltip>
               <div class="add-link">
-                <a
+                <links
                   style="margin: 0"
                   @click="addMatchingFunds()"
                   title="Add matching funds"
                 >
                   <img style="width: 16px" src="@/assets/add.svg" />
-                </a>
-                <a
+                </links>
+                <links
```

Comment:
I think this is causing some errors because there is no "to", therefore its being sent as undefined and causing that error in the router-link